src/components/header/HeaderComponent.tsx

```diff
@@ -11,11 +11,11 @@ import computeStyleSheet from './HeaderComponentStyles';
 export interface Props extends BaseProps {
   title: string;
   subTitle?: string;
-  leftAction?: () => void;
+  leftAction?: () => boolean;
   leftActionIcon?: string;
   leftActionIconType?: IconType;
   hideHomeAction?: boolean;
-  rightAction?: () => void;
+  rightAction?: () => boolean;
```

Comment:
Why changing it to a boolean?

Reply:
It's the type expected by react-native BackHandler.addEventListener() for that callback. 
If you run `npm run tsc`, you have that type mismatch in the list.

Reply:
Okay